connector-setup/steps/configureConnection.js

```diff
@@ -20,49 +23,61 @@ var getCurrentThumbprint = function (workingPath) {
     return thumbprint.calculate(pemToCert(nconf.get('AUTH_CERT')));
   }
 
-  var cert = pemToCert(fs.readFileSync(path.join(workingPath, 'certs', 'cert.pem')).toString());
+  var cert = pemToCert(
+    fs.readFileSync(path.join(workingPath, 'certs', 'cert.pem')).toString()
+  );
   return thumbprint.calculate(cert);
 };
 
 module.exports = function (program, workingPath, connectionInfo, ticket, cb) {
-  var serverUrl = nconf.get('SERVER_URL') ||
-                  ('http://' + os.hostname() + ':' + (nconf.get('PORT') || 4000));
+  var serverUrl =
+    nconf.get('SERVER_URL') ||
+    'http://' + os.hostname() + ':' + (nconf.get('PORT') || 4000);
 
   var signInEndpoint = urlJoin(serverUrl, '/wsfed');
-  var pem = nconf.get('AUTH_CERT') || fs.readFileSync(path.join(workingPath, 'certs', 'cert.pem')).toString();
+  var pem =
+    nconf.get('AUTH_CERT') ||
+    fs.readFileSync(path.join(workingPath, 'certs', 'cert.pem')).toString();
   var cert = pemToCert(pem);
 
-  console.log(('Configuring connection ' + connectionInfo.connectionName + '.').yellow);
+  console.log(
+    ('Configuring connection ' + connectionInfo.connectionName + '.').yellow
+  );
   console.log(' > Posting certificates and signInEndpoint: ' + signInEndpoint);
 
-  request.post({
-    url: ticket,
-    json: {
-      certs:          [cert],
+  axios
+    .post(ticket, {
+      certs: [cert],
       signInEndpoint: signInEndpoint,
-      agentMode:      nconf.get('AGENT_MODE'),
-      agentVersion:   require('../../package').version
-    }
-  }, function (err, response, body) {
-    if (err) {
+      agentMode: nconf.get('AGENT_MODE'),
+      agentVersion: require('../../package').version,
+    })
+    .then((response) => {
+      nconf.set('SERVER_URL', serverUrl);
+      nconf.set('LAST_SENT_THUMBPRINT', getCurrentThumbprint(workingPath));
+      nconf.set('TENANT_SIGNING_KEY', response.data.signingKey || '');
+
+      console.log(
```

Comment:
is this a new console log thats added?

Reply:
No this is from below https://github.com/auth0/ad-ldap-connector/pull/249/files#diff-ddc0f03695a62f6303846c3dd87809c9035df116b5d81e7f143472b0067fe901L65 -- some of this stuff was moved around because of how error handling works in `axios`